### PR TITLE
What individuals can accept

### PR DESCRIPTION
**Pull request by samsont**
broadcaster accepts no video  or audio.
watchers accept no audio.

some re-factoring.

### CODE REVIEW

File: ng-simple-webrtc.js [hunk 1]
```diff
@@ -214,32 +227,23 @@
                 maxWidth: minWidth
               };
             }
+            return webrtcOptions;
+          }
 
-            function displayVideoResolution() {
```
Comment (bahmutov) [inline ng-simple-webrtc.js:218]: why did you remove this method?

**Comment by bahmutov** [comment]
published as version 0.21.0

File: ng-simple-webrtc.js (hunk 1 above)
Comment (samsont) [inline ng-simple-webrtc.js:218]: Feel free to put it back.

When we started using 'onloadedmetadeta', there was a bunch of code in
show-whiteboard-simple-webrtc.js that could be removed.
I was pruned a bunch of code -- I could easily have overdone it.

 Thanks!

-- Samson
